packages/web/src/common/store/account/sagas.js

```diff
@@ -197,31 +200,14 @@ export function* fetchAccountAsync({ isSignUp = false }) {
 
 export function* fetchLocalAccountAsync() {
   const localStorage = yield getContext('localStorage')
-  const audiusBackendInstance = yield getContext('audiusBackendInstance')
 
   yield put(accountActions.fetchAccountRequested())
 
-  const audiusLibs = yield call([
-    audiusBackendInstance,
-    audiusBackendInstance.getAudiusLibs
-  ])
-  const wallet = yield call([
-    audiusLibs.web3Manager,
-    audiusLibs.web3Manager.getWalletAddress
-  ])
   const cachedAccount = yield call([localStorage, 'getAudiusAccount'])
   const cachedAccountUser = yield call([localStorage, 'getAudiusAccountUser'])
   const currentUserExists = yield call([localStorage, 'getCurrentUserExists'])
 
-  const walletMatches =
-    wallet?.toLowerCase() === cachedAccountUser?.wallet?.toLowerCase()
-
-  if (
-    cachedAccount &&
-    cachedAccountUser &&
-    !cachedAccountUser.is_deactivated &&
-    walletMatches
-  ) {
+  if (cachedAccount && cachedAccountUser && !cachedAccountUser.is_deactivated) {
```

Comment:
If any of these fail would it make sense to then do the libs calls above? So we get fast initial for most of it with a fallback?

Reply:
the wallet could only match in the first place if there is a `cachedAccount`

instead i think our long term move here is to get libs init to be faster (remove libs in favor of sdk and kill userStateManager) and then maybe just drop the cached user logic.

if we only wait on the /account endpoint to come back, it should be fast enough to skip this whole caching optimization

Reply:
that general plan makes sense! would be interested to see if account endpoint ends up being fast enough